pkg/pipeline/ingest/ingest_kafka.go

```diff
@@ -74,14 +76,61 @@ func (ingestK *ingestKafka) kafkaListener() {
 				log.Errorln(err)
 			}
 			log.Debugf("string(kafkaMessage) = %s\n", string(kafkaMessage.Value))
-			if len(kafkaMessage.Value) > 0 {
+			messageLen := len(kafkaMessage.Value)
+			if messageLen > 0 {
+				trafficLabels := prometheus.Labels{
+					"type":       ingestK.kafkaParams.Decoder.Type,
+					"remote_ip":  ingestK.kafkaParams.Brokers[0],
+					"local_ip":   "0.0.0.0",
+					"local_port": "0",
+				}
+				flowTrafficBytesSum.With(trafficLabels).Observe(float64(messageLen))
 				ingestK.in <- string(kafkaMessage.Value)
 			}
 		}
 	}()
 
 }
 
+func processRecordDelay(record config.GenericMap) {
+	TimeFlowEndInterface, ok := record["TimeFlowEnd"]
+	if !ok {
+		flowErrors.With(prometheus.Labels{"router": "", "error": "No TimeFlowEnd found"}).Inc()
+		return
+	} else {
+		fmt.Println(TimeFlowEndInterface)
```

Comment:
Do you really want to print to stdout for each end of flow? or is this only for debug purposes?

Reply:
Definitly not, this is a debug trace that I forgot to remove.
Thanks